vnext/Microsoft.ReactNative/Views/ViewManagerBase.cpp

```diff
@@ -319,6 +319,7 @@ void ViewManagerBase::NotifyUnimplementedProperty(
     ShadowNodeBase *nodeToUpdate,
     const std::string &propertyName,
     const winrt::Microsoft::ReactNative::JSValue &value) {
+#ifndef ARCHONRNW
```

Comment:
This looks unrelated?
Separately we should probably look at disabling this by default --  it does produce a lot of debug spew which most apps are not going to want clogging up their debug output.

Reply:
Yeah, my bad. Was testing if disabling those logs would make the debug build any faster and this leaked in 😬

Reply:
Fixed :)